src/main/java/lesson12Names/NameChanger.java

```diff
@@ -0,0 +1,28 @@
+package lesson12Names;
+
+import java.util.*;
+
+public class NameChanger {
+
+    public static void main(String[] args) {
+
+        List<String> names = Arrays.asList
+                ("Anna", "Katarzyna", "Tymon", "Tadeusz", "Anna", "Karolina", "Dorota", "Piotr");
+
+        Set<String> namesSet = new HashSet<>(names);
```

Comment:
Można jeszcze zastosować names.stream().distinct().toList(); - odfiltrue to duplikaty